fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -348,7 +360,7 @@ class ReadIteratorPool {
 	}
 
 	// Called on every read operation.
-	ReadIterator getIterator() {
+	ReadIterator getIterator(const KeyRange& range) {
```

Comment:
Please leave a comments about Iterator bound is not set if `ROCKSDB_READ_RANGE_REUSE_ITERATORS` is enabled.